bootcamp/tutorials/integration/openai_agents_milvus.ipynb

```diff
@@ -0,0 +1,490 @@
+{
```

Comment:
<p>Before this code snippet, can you briefly mention the words "Before running milvus client, please start the milvus docker server first. Refer to <a href="https://milvus.io/docs/install_standalone-docker-compose.md%22" target="_blank">https://milvus.io/docs/install_standalone-docker-compose.md" ?</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/milvus-io/bootcamp/pull/1505/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='4'/>

Reply:
<p>Good catch, added some text about it </p>